BambooTracker/gui/order_list_editor/order_list_panel.cpp

```diff
@@ -105,7 +105,7 @@ OrderListPanel::OrderListPanel(QWidget *parent)
 	headerFont_.setPointSize(10);
 	rowFont_ = QFont("Monospace", 10);
 	rowFont_.setStyleHint(QFont::TypeWriter);
-	rowFont_.setStyleStrategy(QFont::ForceIntegerMetrics);
+	rowFont_.setStyleStrategy(QFont::PreferMatch);
```

Comment:
`QFont::ForceIntegerMetrics` was removed in Qt6. I couldn't find a perfect 1:1 replacement from the docs so I picked `QFont::PreferMatch`. If the choice of `QFont::ForceIntegerMetrics` was a deliberate one and `QFont::PreferMatch` breaks something visually, please let me know.